lib/game.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../lib/board.rb'
+class Game
+  attr_accessor :board_tic, :player1, :player2, :winning_moves
+
+  def initialize(player1, player2)
+    @board_tic = Board.new
+    @player1 = player1
+    @player2 = player2
+    @winning_moves = [
+      [1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7],
+      [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]
+    ]
+  end
+
+  def mark_board(player)
+    return if @board_tic.marked?(player)
```

Comment:
This is not the best way to avoid selecting the same cell. I can mark select the same cell any time I want but I get no warning.
Check how you are switching turns. I can type anything (even if it's wrong) and my turn ends instead of giving me notice and allowing me to select again.